helm/values.yaml

```diff
@@ -112,14 +112,14 @@ metricAnomalyDetectorConfig:
     host: attribute-service
     port: 9012
   queueConfig:
-    bootstrapServers: bootstrap
-    schemaRegistryUrl: localhost
+    bootstrapServers: "bootstrap:9092"
+    schemaRegistryUrl: "http://schema-registry-service:8081"
     consumer:
-      keyDeserializer: ""
-      valueDeserializer: ""
+      keyDeserializer: "io.confluent.kafka.streams.serdes.avro.SpecificAvroDeserializer"
```

Comment:
I've asked this in a bunch of various PRs, mostly unrelated to this - why do we even expose this for configuration?  Our code relies on a specific deserializer because it knows the class it expects, right? Wouldn't it be better to just declare this in code then?

Reply:
This allows for the service to work without schema registry

Reply:
Interesting - so when using schema registry we use a different avro deserializer?